### PR TITLE
line and area on mapbox

### PR DESCRIPTION
**Pull request by Mahdis-z**
Doc upgrade checklist:

- [ ] random seed is set if using random data
- [ ] file has been moved from `unconverted/x/y.md` to `x/y.md`
- [ ] old boilerplate at top and bottom of file has been removed
- [x] Every example is independently runnable and is optimized for short line count
- [x] no more `plot()` or `iplot()`
- [ ] `graph_objs` has been renamed to `graph_objects`
- [x] `fig = <something>` call is high up in each example
- [x] minimal creation of intermediate `trace` objects
- [x] liberal use of `add_trace` and `update_layout`
- [x] `fig.show()` at the end of each example
- [ ] `px` example at the top if appropriate
- [x] minimize usage of hex codes for colors in favour of those in https://github.com/plotly/plotly.py-docs/issues/14

### CODE REVIEW

File: python/Line-on-Mapbox.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+      Plotly.
+    display_as: maps
+    has_thumbnail: true
+    ipynb: ~notebook_demo/56
+    language: python
+    layout: user-guide
+    name: Lines on Maps
+    order: 1
+    page_type: example_index
+    permalink: python/lines-on-maps/
```
Comment (nicolaskruchten) [inline python/Line-on-Mapbox.md]: You can't use this permalink, as this URL already exists: https://plot.ly/python/lines-on-maps/

File: python/Line-on-Mapbox.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+fig.update_layout(
+    mapbox = {
+        'accesstoken': token,
+        'style': "outdoors", 'zoom': 0.7},
+    showlegend = False)
+
+fig.show()
+```
+### Contour Line on Glob
```
Comment (nicolaskruchten) [inline python/Line-on-Mapbox.md]: this is not a mapbox example and should be removed.

File: python/Line-on-Mapbox.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+fig.update_layout(margin ={'l':0,'t':0,'b':0,'r':0},
+    mapbox = {'center': {'lon': 10, 'lat': 10},
+        'style': "stamen-terrain", 
+        'center': {'lon': -20, 'lat': -20}, 
+        'zoom': 1})
+
+fig.show()
+```
+
+### Set Marker Symbols
```
Comment (nicolaskruchten) [inline python/Line-on-Mapbox.md]: this probably belongs in the mapbox scatter page, not the mapbox line page.

File: python/Mapbox-area.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+
+### Mapbox Access Token
+
+To plot on Mapbox maps with Plotly you *may* need a Mapbox account and a public [Mapbox Access Token](https://www.mapbox.com/studio). See our [Mapbox Map Layers](/python/mapbox-layers/) documentation for more information.
+
+
+### How to Show an Area on a Map 
+
+There are three different ways to show an area in a mapbox: 
+- Set `fill` attribute to 'toself'
```
Comment (nicolaskruchten) [inline python/Mapbox-area.md]: mention that this is using `scattermapbox` traces

File: python/Line-on-Mapbox.md (hunk 1 above)
Comment (nicolaskruchten) [reply python/Line-on-Mapbox.md]: the filename is OK to use as a permalink, if you lowercase it :)

File: python/Mapbox-area.md [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+      Plotly.
+    display_as: maps
+    has_thumbnail: true
+    ipynb: ~notebook_demo/56
+    language: python
+    layout: user-guide
+    name: Filled Area on Maps
+    order: 1
+    page_type: example_index
+    permalink: python/filled-area-on-mapbox/
```
Comment (nicolaskruchten) [inline python/Mapbox-area.md]: please use this permalink as the filename.

File: python/Line-on-Mapbox.md (hunk 3 above)
Comment (nicolaskruchten) [reply python/Line-on-Mapbox.md]: here: https://plot.ly/python/scattermapbox/

File: python/Mapbox-area.md [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+
+fig.update_layout(
+    mapbox = {'style': "stamen-terrain", 'center': {'lon': 30, 'lat': 30}, 'zoom': 2},
+    showlegend = False,
+    margin = {'l':0, 'r':0, 'b':0, 't':0})
+
+fig.show()
+```
+
+### Use the Coresponding Geojson
```
Comment (nicolaskruchten) [inline python/Mapbox-area.md]: 2 Rs in corresponding

**Comment by nicolaskruchten** [comment]
what's the difference between this PR and https://github.com/plotly/plotly.py-docs/pull/125 ?

**Comment by Mahdis-z** [comment]
> what's the difference between this PR and #125 ?

area-line is the branch name for js docs as well. So switching to the area-line branch I couldn't see the changes in local repo! I'll close this pr once made the revisions.

**Comment by nicolaskruchten** [comment]
replaced with #125

**Pull request closed without merging**